### PR TITLE
Vellum case management migration: ability to add domains by email

### PR DESCRIPTION
**Pull request by orangejenny**
screen shot 2016-07-26 at 5 53 56 pm

@benrudolph 
fyi @krsdimagi

### CODE REVIEW

File: corehq/apps/hqadmin/views.py [hunk 1]
```diff
@@ -488,50 +489,92 @@ def page_context(self):
...
+                    if domain not in errors:
+                        if migration.notes:
+                            migration.notes = migration.notes + '; '
+                        migration.notes = migration.notes + "successfully migrated {} domains".format(app_count)
+                        successes.add(domain)
+                    migration.migrated = datetime.now()
+                    migration.save()
+            if len(successes):
+                messages.success(request, "Succeeded with the following {} domains: {}".format(
+                                            len(successes), ", ".join(successes)))
```
Comment (dimagimon) [inline corehq/apps/hqadmin/views.py]: E126 continuation line over-indented for hanging indent

File: corehq/apps/hqadmin/views.py [hunk 2]
```diff
@@ -488,50 +489,92 @@ def page_context(self):
...
+            if len(errors):
+                messages.error(request, "Errors in the following {} domains: {}".format(
+                                            len(errors), ", ".join(errors)))
```
Comment (dimagimon) [inline corehq/apps/hqadmin/views.py]: E126 continuation line over-indented for hanging indent

File: corehq/apps/hqadmin/views.py [hunk 3]
```diff
@@ -488,50 +489,92 @@ def page_context(self):
...
-                m.migrated = datetime.now()
-                m.save()
-        if len(successes):
-            messages.success(request, "Succeeded with the following {} domains: {}".format(
-                                        len(successes), ", ".join(successes)))
-        if len(errors):
-            messages.error(request, "Errors in the following {} domains: {}".format(
-                                        len(errors), ", ".join(errors)))
+        # Form submission
+        if action == 'add':
```
Comment (benrudolph) [inline corehq/apps/hqadmin/views.py]: this logic is getting pretty complex, if important, may be worth adding a test

**Comment by benrudolph** [comment]
👍 dimagimon

**Comment by orangejenny** [comment]
linted

File: corehq/apps/hqadmin/views.py (hunk 3 above)
Comment (orangejenny) [inline corehq/apps/hqadmin/views.py]: Hmm, not a bad point. Will consider adding tests if I do much more dev on this page. I'm hoping that this'll be the last significant PR (famous last words...).

**Pull request merged**